packages/core/src/testInteg/shared/utilities/workspaceUtils.test.ts

```diff
@@ -297,6 +297,12 @@ describe('collectFiles', function () {
             ] satisfies typeof result,
             result
         )
+
+        assertTelemetry('function_call', {
+            totalFiles: 8,
+            totalFileSizeInMB: 0.0002593994140625,
```

Comment:
Is this value fixed? I.e. it is generated from constant test data?

Reply:
Yeah its generated from the size of the files